orion/handlers/utils.go

```diff
@@ -117,6 +117,8 @@ func getInterceptors(svc interface{}, config CommonConfig, middlewares []string)
 		opts = append(opts, interceptors.DefaultInterceptors()...)
 	}
 
+	opts = append(opts, interceptors.GRPCLoggingInterceptor(config.EnableGRPCRequestLog, config.EnableGRPCResponseLog))
```

Comment:
Is there a reason `GRPCLoggingInterceptor` is adding this way and not in `DefaultInterceptors`?

Reply:
In my opinion having a `DefaultInterceptors` is not a good idea - it breaks the single responsibility principle and from the method name it is difficult to guess what it does, what it's contract

how do you think @praveensanap

Reply:
1. The `if !config.DisableDefaultInterceptors` check doesn't disable for your interceptor. Your interceptor is included even if a service is explicitly trying to disable all default interceptors.
2. Specifying all the default interceptors in `DefaultInterceptors` makes it easier to inspect what the default interceptors are. Whereas with this approach, some default interceptors are specified in one place, and some are specified in another place. It's overall less readable.